src/components/shared/SearchInput.js

```diff
@@ -1,33 +1,17 @@
-const React = require("react");
-const { DOM: dom, PropTypes } = React;
-const classnames = require("classnames");
-const Svg = require("./Svg");
-const CloseButton = require("./Button/Close").default;
+import { DOM as dom, PropTypes, Component } from "react";
+import Svg from "./Svg";
+import classnames from "classnames";
+import CloseButton from "./Button/Close";
+import "./SearchInput.css";
 
-require("./SearchInput.css");
+class SearchInput extends Component {
```

Comment:
I think we can change this to `export default class SearchInput extends Component {` and remove the last line (which has export statement).